ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml

```diff
@@ -38,7 +38,7 @@ tests:
   cron: 15 * * * *
   steps:
     env:
-      OCP_RELEASES: 4.12 4.13 4.14 4.15 4.16 4.17
+      OCP_RELEASES: 4.12 4.13 4.14 4.15 4.16 4.17 4.18
```

Comment:
@LuboTerifaj if you want to enable 4.18 in QE job controller in this PR, we need to merge https://github.com/openshift/release-tests/pull/423 first, otherwise, job controller cannot find job config from test job registry

Reply:
@rioliu-rh sure, can we merge both pull-requests at the same time? Splitting the pull-requests is a little bit redundant work with each release stream.

Reply:
as I know, it's impossible to merge 2 PRs at the same time.

Reply:
Of course it's not :D The question is, what is the benefit of having this pull-request split. If you merge dependent pull-requests within reasonable time, meaning a few seconds, or in the worst case a few minutes, I don't see any issue there. What are the benefits having separate pull-requests?

Reply:
maybe you misunderstand my point, I'm not asking to split the PR. I'm talking about the relationship/dependency b/w different PRs.

Reply:
I opened the pull-requests together. They are linked between each other since beginning. They have the same name, the same jira. They of course should be merged both together, as one depends on the other and vice versa. I would say, we can consider this discussion as resolved.